packages/babel-cli/src/babel/index.js

```diff
@@ -126,4 +128,51 @@ if (commander.outDir) {
   fn = require("./file");
 }
 
+if(commander.settings) {
+  let fileOptionsManager = require("babel-core").File,
+      allOptions = [];
+  let generalOptions = new fileOptionsManager( { filename: "unknown" } ).opts;
```

Comment:
Is `"unknown"` a magic string or just something that happens to work? If it's a magic string can we reference it? If not, maybe we should go with something that's less likely to be used in the real world?